eventlet/patcher.py

```diff
@@ -1,5 +1,6 @@
 import imp
 import sys
+from importlib import import_module
```

Comment:
Only module imports are fine. `from module import function` is a recipe for trouble and is prohibited unless absolutely necessary.

Reply:
Sorry, will fix that importlib import and usage.

I'm okay with a more sweeping change to use importlib. My thought was to minimize risk. However, I do realize that unless all `__import__()` calls are changed, one application may work in a frozen environment while another, that uses more of eventlet, may not.